demos/supabase-anonymous-auth/lib/widgets/status_app_bar.dart

```diff
@@ -1,5 +1,6 @@
 import 'dart:async';
 
+import 'package:flutter/foundation.dart';
```

Comment:
Based on all the repeated changes should we not just create a common component for that status bar and then have all the demos use it?

Reply:
Technically the status bar is slightly different between demos (one contains search), but the status icon is the same and could be re-used.

I'm a little conflicted on this one: One the one side a common package would allow us to re-use code between the demos, but on the other side it makes it more difficult for others to copy and modify a demo.

What I think would actually be best is to reduce the scope of some of the demos (specifically supabase-anonymous-auth and supabase-edge-auth) to not contain the entire todolist app, but rather just a minimal amount of code to show how to use custom auth functions.

Reply:
Yeah that makes sense. I left the status bar in the auth demos because of the status icon mainly but don't think it's really necessary